docs/topics/mta-web-run-assessment.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-run-assessment{context}"]
+== Running an assessment on an application
+
```

Comment:
We need a description here -- and what's not clear to me is if we run can an assessment on an archetype or we always run assessments on applications, which get built up into archetypes.